examples/android/helloworld/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
How did you notice this? This shouldn't actually do anything because "build" is ignored at the root.